stack-graphs/src/storage.rs

```diff
@@ -321,7 +322,8 @@ impl SQLiteWriter {
         let mut stmt =
             conn.prepare_cached("INSERT INTO graphs (file, tag, value) VALUES (?, ?, ?)")?;
         let graph = serde::StackGraph::from_graph_filter(graph, &FileFilter(file));
-        stmt.execute((file_str, tag, &postcard::to_stdvec(&graph)?))?;
+        let serialized = bincode::encode_to_vec(&graph, config::standard())?;
```

Comment:
Perhaps we can define a module level `CONFIG` variable/constant that is used everywhere?